dm/task-configuration-file-full.md

```diff
@@ -111,6 +111,16 @@ loaders:
     pool-size: 16                    # The number of threads that concurrently execute dumped SQL files in the load processing unit (16 by default). When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.
     dir: "./dumped_data"             # The directory that the load processing unit reads from and the dump processing unit outputs SQL files to ("./dumped_data" by default). The directories for different tasks of the same instance must be different.
 
+# The import mode during the full import phase. You can set it to the following:
+# - "sql" (default). Use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) TiDB-backend mode to import data.
+# - "loader". Use Loader mode to import data. This mode is only for compatibility with features that TiDB Lightning does not support yet. It will be deprecated in the future.
+import-mode: "sql"
+#  Methods to resolve conflicts during the full import phase. You can set it to the following:
+# - "replace" (default). Only supports the import mode "sql". In this method, it uses the new data to replace the existing data.
+# - "ignore". Only supports the import mode "sql". It keeps the existing data, and ignores the new data.
+# - "error". Only supports the import mode "loader". It reports errors when inserting duplicated data, and then stops the replication task.
+on-dupicate: "replace"
```

Comment:
Please indent as the Chinese version does.

Reply:
Done